TDAT: public void test get object timeout reap ( ) throws exception {  <NL> server . prepare and commit ( tid ) ;  <NL> server . shutdown ( ) ;  <NL> props . set property ( data store net package + " . server . reap . delay " , " 50 " ) ;  <NL> server = get data store server ( ) ;  <NL> tid = server . create transaction ( 100 ) ;  <NL> server . set binding ( tid , " dummy " , oid ) ;  <NL> thread . sleep ( 200 ) ;  <NL> try {  <NL> server . get binding ( tid , " dummy " ) ;  <NL> fail ( " expected transaction timeout exception " ) ;  <NL>  } catch ( transaction not active exception e ) {  <NL> system . err . println ( e ) ;  <NL>  }  <NL> try {  <NL> server . abort ( tid ) ;  <NL> fail ( " expected transaction not active exception " ) ;  <NL>  } catch ( transaction not active exception e ) {  <NL> system . err . println ( e ) ;  <NL> tid = - 1 ;  <NL>  }  <NL>  }  COM: <s> test that the standard transaction timeout gets applied by the </s>